client/lxd_containers.go

```diff
@@ -207,8 +207,10 @@ func (r *ProtocolLXD) tryCreateContainer(req api.ContainersPost, urls []string)
 
 			rop.targetOp = op
 
-			for _, handler := range rop.handlers {
-				_, _ = rop.targetOp.AddHandler(handler)
+			if r.supportsAuthentication {
+				for _, handler := range rop.handlers {
+					_, _ = rop.targetOp.AddHandler(handler)
```

Comment:
Should we be ignoring the error returned from AddHandler?

Reply:
I've seen this sort of thing all over the place wherever we try to set up an events listener with `getEvents` or `AddHandler` and I didn't understand the reason.